openml-utils/src/main/java/com/feedzai/openml/util/validate/ClassificationValidationUtils.java

```diff
@@ -70,4 +76,33 @@ public static void validateClassificationModel(final DatasetSchema schema, final
             throw new ModelLoadingException(msg, e);
         }
     }
+
+    /**
+     * Validates that the model to load can be used with the given parameters.
+     *
+     * @param modelLoader An instance responsible for instantiating models.
+     * @param modelPath   Path of the model.
+     * @param schema      Schema of the model.
+     * @param params      Collection of parameters and the corresponding values.
+     * @throws ModelLoadingException If the path and/or schema are not valid.
+     * @since @@@feedzai.next.release@@@
+     */
+    public static void validateParamsModelToLoad(final MachineLearningModelLoader modelLoader,
+                                                 final Path modelPath,
+                                                 final DatasetSchema schema,
+                                                 final Map<String, String> params) throws ModelLoadingException {
+        logger.info("Validating the parameters of a model to load");
+        final List<ParamValidationError> validationErrors = modelLoader.validateForLoad(
+                modelPath,
+                schema,
+                params
+        );
+        if (!validationErrors.isEmpty()) {
+            final String errorMsg = validationErrors.stream()
+                    .map(ParamValidationError::getMessage)
+                    .collect(Collectors.joining(","));
+            throw new ModelLoadingException(errorMsg);
+        }
+        logger.info("Parameters of the model to load successfully validated");
```

Comment:
Maybe add the modelPath to the info log so that we can track which one it is.
Also I think this is more of a debug than info.